projects/pt1/python/torch_mlir_e2e_test/configs/onnx_backend.py

```diff
@@ -92,7 +92,7 @@ def _module_lowering(
     # Lower from ONNX to Torch
     run_pipeline_with_repro_report(
         torch_mod,
-        f"builtin.module(func.func({ONNX_TO_TORCH_FUNC_PIPELINE}))",
+        f"builtin.module(inline, func.func({ONNX_TO_TORCH_FUNC_PIPELINE}))",
```

Comment:
This needs a comment. Something like:

```
# ONNX import produces multiple functions (for custom operators, etc), but the reference backend does not support calls currently. Therefore, since this is just for testing, we unconditionally inline.
```

Reply:
Right, I was wondering about adding a comment here. How does this sound?

> The importer may produce MLIR functions corresponding to ONNX functions. These need to be inlined before conversion so it won't choke on them.

Reply:
I added a comment, I hope it's satisfactory.